### PR TITLE
Tbs enable auto updates details

### PR DESCRIPTION
**Pull request by tylerphelan**
Which other branches should this be merged with (if any)?

1.0

- Update install instructions for automatic dependency updates in TBS component install docs
- Update install instructions for automatic dependency updates in full and lite profiles

This is a consolidation of https://github.com/pivotal/docs-tap/pull/842 and https://github.com/pivotal/docs-tap/pull/843 as well as adding info for lite profile. These PRs can be closed in favor of this one.

### CODE REVIEW

File: install-components.md [hunk 1]
```diff
@@ -1075,7 +1075,7 @@ To install Tanzu Build Service using the Tanzu CLI:
     tanzunet_username: TANZUNET-USERNAME
     tanzunet_password: TANZUNET-PASSWORD
     descriptor_name: DESCRIPTOR-NAME
-    enable_automatic_dependency_updates: true
+    enable_automatic_dependency_updates: "true-or-false" # Optional
```
Comment (matthewmcnew) [inline install-components.md:1078]: The quotes here kind of look like the value "true-or-false" is expected. I am not sure if this is better but, maybe?
```suggestion
    enable_automatic_dependency_updates:  # Optionally either "true" or "false"
```

Comment (tylerphelan) [reply install-components.md]: agreed except that true/false shouldn't be quoted I think that could be confusing

**Comment by a-peek4** [comment]
@tylerphelan Please ping #tap-docs when you're ready to take this out of draft state so we can merge

**Comment by tylerphelan** [comment]
@a-peek4 ready for review

**Comment by RichardJJG** [comment]
Thanks, I've merged and will edit and cherry-pick now.

**Pull request merged**